src/www/pages/styles/utility-classes/index.njk

```diff
@@ -194,6 +210,21 @@ Where <code class="wmnds-website-inline-code">{size}</code> is one of:
 </pre>
 
 
+<hr>
+{# Background #}
+<h2>Background Colour</h2>
+<p>
+  Changes background to the intended colour.
+</p>
 
+<pre class="wmnds-col-1 wmnds-col-md-1-2">
+  <code class="wmnds-show-more-ignore scss" tabindex="0">
+    .bg-default
+    .bg-white
+    .bg-grey
+    .bg-yellow
+    .bg-transparent
```

Comment:
Can you update the bg names to the same colour names on the [colour palettes page](https://deploy-preview-386--wmnetwork.netlify.app/styles/colour-palettes/) so they are in sync.

An example is changing bg-yellow to bg-planned-disruption.